cmd/utils/flags.go

```diff
@@ -513,6 +520,35 @@ func MakeAccountManager(ctx *cli.Context) *accounts.Manager {
 	return accounts.NewManager(ks)
 }
 
+// MakeExtra resolves extradata for the miner from a flag or returns a default.
+func makeExtra(ctx *cli.Context) []byte {
+	if ctx.IsSet(ExtraDataFlag.Name) {
```

Comment:
@bas-vk I think it was my suggestion. Didn't know there's a global version.
On Sep 1, 2015 10:04 PM, "bas-vk" <notifications@github.com> wrote:

> In cmd/utils/flags.go
> <https://github.com/ethereum/go-ethereum/pull/1744#discussion_r38458194>:
>
> > @@ -513,6 +520,35 @@ func MakeAccountManager(ctx *cli.Context) *accounts.Manager {
> >  	return accounts.NewManager(ks)
> >  }
> >
> > +// MakeExtra resolves extradata for the miner from a flag or returns a default.
> > +func makeExtra(ctx *cli.Context) []byte {
> > +	if ctx.IsSet(ExtraDataFlag.Name) {
>
> Why did you choose IsSet over GlobalIsSet?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ethereum/go-ethereum/pull/1744/files#r38458194>.
>